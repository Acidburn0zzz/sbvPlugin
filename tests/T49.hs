../Data/SBV/Plugin/Examples/BitTricks.hs