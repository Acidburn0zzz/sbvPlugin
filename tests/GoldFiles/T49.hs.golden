
[SBV] tests/T49.hs:37:1-14 Proving "fastMinCorrect", using Z3.
[Z3] Q.E.D.

[SBV] tests/T49.hs:44:1-14 Proving "fastMaxCorrect", using Z3.
[Z3] Q.E.D.

[SBV] tests/T49.hs:51:1-20 Proving "oppositeSignsCorrect", using Z3.
[Z3] Q.E.D.

[SBV] tests/T49.hs:58:1-26 Proving "conditionalSetClearCorrect", using Z3.
[Z3] Q.E.D.

[SBV] tests/T49.hs:66:1-17 Proving "powerOfTwoCorrect", using Z3.
[Z3] Q.E.D.
